tealiumlibrary/src/main/java/com/tealium/core/consent/ConsentManagerConstants.kt

```diff
@@ -12,3 +12,8 @@ object ConsentManagerConstants {
     const val GRANT_PARTIAL_CONSENT = "grant_partial_consent"
     const val DECLINE_CONSENT = "decline_consent"
 }
+
+object ConsentManagerSPKey {
```

Comment:
not sure this needs to be on a whole separate class - no reason why it can't go on the ConsentManagerConstants above.

Reply:
Was just following convention in some other modules - LifecycleSPKey, for example